doc/reference/storage_lvm.md

```diff
@@ -46,6 +46,7 @@ lvm.vg.force\_reuse           | bool                          | false
 lvm.vg\_name                  | string                        | name of the pool                        | Name of the volume group to create
 rsync.bwlimit                 | string                        | 0 (no limit)                            | The upper limit to be placed on the socket I/O when rsync must be used to transfer storage entities
 rsync.compression             | bool                          | true                                    | Whether to use compression while migrating storage pools
+size                          | string                        | 0                                       | Size/quota of the storage pool in bytes (suffixes supported, currently valid for loop-based pools and ZFS)
```

Comment:
Should we make it clearer here that this can only be specified at creation time?
And is the comment about loop-based pools and ZFS correct? It seems the option exists for Btrfs, LVM and ZFS, but not for dir.

Reply:
I agree saying its only used at create time makes sense.
The statement about loop-based pools covers Btrfs, LVM and ZFS.
Im interested what the "and ZFS" bit is getting at though, as it suggests `size` for ZFS pools can be used when not using loop-based pools.

Reply:
But isn't dir loop-based as well? 
https://linuxcontainers.org/lxd/docs/stable-5.0/explanation/storage/#data-storage-location

Reply:
I checked, no dir does not support loop files (there's no need for that).

Reply:
Check ZFS too, the `size` setting is only used for loop pools, not ZFS pools using an existing device/ZFS pool.

Reply:
So can just say:

> Size of the storage pool in bytes when creating loop-based pools (suffixes supported)

Reply:
I'd like to change the validation on pool creation so that `size` is rejected if provided when not creating a loop pool, as right now it just sets any `size` provided to empty string when creating a non-loop pool.

Reply:
Yeah source is create time only too.